.travis.yml

```diff
@@ -21,6 +21,7 @@ script:
 os:
   - linux
   - osx
+osx_image: xcode6.4
```

Comment:
Oh this actually ended up being upgraded as part of https://github.com/rust-lang/libc/pull/307 as well, wanna back this out for now as that may land ahead of this?